lib/metalsmith.js

```diff
@@ -10,6 +10,7 @@ const sass = require('metalsmith-sass')             // convert Sass files to CSS
 
 const debug = require('./debug')                    // debug plugin
 const navigation = require('./navigation.js')       // navigation plugin
+const env = require('metalsmith-env');              // environment vars plugin
```

Comment:
😨 `;`

we should probably remove it at some point 😄

Reply:
OH NOES

Reply:
https://github.com/alphagov/govuk-design-system/pull/144